docs/docs/cmd/pa/app/app-export.md

```diff
@@ -0,0 +1,68 @@
+# pa app export
+
+Exports the specified Power App
+
+## Usage
+
+```sh
+m365 pa app export [options]
+```
+
+## Options
+
+`-i, --id <id>`
+: The id of the Power App to export
+
+`-e, --environment <environment>`
+: The name of the environment for which to export the app
+
+`-n, --packageDisplayName [packageDisplayName]`
+: The display name to use in the exported package
+
+`-d, --packageDescription [packageDescription]`
+: The description to use in the exported package
+
+`-c, --packageCreatedBy [packageCreatedBy]`
+: The name of the person to be used as the creator of the exported package
+
+`-s, --packageSourceEnvironment [packageSourceEnvironment]`
+: The name of the source environment from which the exported package was taken
+
+`-p, --path [path]`
+: The path to save the exported package to
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Export the specified Power App as a ZIP file
```

Comment:
As lot of options are optional, let's add another example above this one with minimal options.